ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -225,6 +225,9 @@ export class MultiSigTransactionPayload {
   }
 
   static deserialize(deserializer: Deserializer): MultiSigTransactionPayload {
+    // TODO: Support other types of payload beside EntryFunction.
+    // This is the enum value indicating which type of payload the multisig tx contains.
+    deserializer.deserializeUleb128AsU32();
     return new MultiSigTransactionPayload(EntryFunction.deserialize(deserializer));
```

Comment:
Is there a reason you don't just deserialize the number and then throw an appropriate error saying it's not an entry function?

Reply:
We don't support a different payload type anyway so it's not possible to have an unexpected type. If user somehow obtains an invalid hex string from somewhere and tries to deserialize it, that'd fail anyway with deserialization error. Not sure how useful it is to just return an error message based on that single enum bit